dali/test/python/test_operator_warp.py

```diff
@@ -56,14 +56,14 @@ def ToCVMatrix(matrix):
   result[1][2] = offset[1] - 0.5
   return result
 
-def CVWarp(output_type, input_type, warp_matrix = None):
+def CVWarp(output_type, input_type, warp_matrix = None, inverted = False):
```

Comment:
`inverted` sounds like `flipped vertically`
```suggestion
def CVWarp(output_type, input_type, warp_matrix = None, inv_map = False):
```

Reply:
done